### DISCUSSION TITLE
Escrever artigos em português ou ingrêis... ?

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#433] [Discussão]
Fala pessoal,

O @andre-mendes, meu "pupilo" no projeto [Mentoria](https://github.com/training-center/mentoria) acabou de me questionar com essa dúvida.

Eis os pontos que levantei:

- A grande maioria dos devs no Brasil nao conseguem ler em inglês, se escrever em inglês, abre o artigo para o mundo mas limita o alcance aqui no Brasil.
- O que estou fazendo é analisando antes de escrever... Se achar que esse assunto ja foi muito abordado na comunidade internacional, mas não tanto aqui no Brasil, escrevo em pt-br. Se for uma coisa genérica, como [meu dia a dia como desenvolvedor](https://medium.com/@lfeh/my-day-to-day-as-front-end-developer-in-2017-6d68b5ac2055), escrevo em inglês.

Mas sinceramente não tenho nenhuma conclusão para esse assunto.

O que pensam?

**Comment by malaquiasdev** [comment]
Por que não fazer os dois?

Quando vc analisar que dá pra escrever em inglês, escreva em pt-br também. Estou avaliando isso também e pretendo fazer isso.

**Comment by hjdesigner** [comment]
Acho legal se for nos dois.

Tipo BR você ajuda a galerinha daqui e inglês vai te ajudar a ser visto no mundo.

**Comment by felipefialho** [comment]
@mmalaquias1 Eu meio que faço. 

Cheguei a deixar a conta do Medium apenas para traduzir meus artigos para inglês. Com o tempo acabei usando ela para artigos mais pessoais também (deixando meu site para artigos mais técnicos).

O problema é... tempo...

Para escrever duas versões do mesmo artigo, bastante tempo e disposição são necessários, e nem sempre é possível.

Mas claro, é uma boa pratica.

**Comment by JoseCage** [comment]
Entre Inglês ou Português em todo caso acredito que o fator decisivo é "Quem vai manter o código", se é de escopo global ou nacional. Pois, se for o caso de um projeto para o Brasil, feito por Brasileiros e que será mantido por brasileiros fica mais em conta escrever mesmo em Português e deixar perceptível para todos sendo que ainda há uma maioria esmagadora de pessoas com dificuldade no Inglês (**Falando do número sem propriedades**, pois não sou brasileiro.. 👶 = :angola: ).

Já quando se trata mesmo de um projeto (principalmente **OpenSource**) o melhor e fazer em inglês dando a possibilidade de maior alcance.

**Comment by hugooliveirad** [comment]
Recentemente experimentei fazendo os dois com o meu artigo sobre [composição de componentes][comp-comp]. A [versão em inglês][comp-comp-en] dele teve muito mais feedback, pois conseguir alcançar mais pessoas.

[comp-comp]: http://hugobessa.com.br/2016/07/01/composicao-componentes/
[comp-comp-en]: http://hugobessa.com.br/2016/07/20/composable-components/

Eu costumava tuitar em inglês também, em [@hugobessaa][twitter], mas voltei a conversar em português, pois quero focar na comunidade local e a grande maioria não tem um nível muito legal de inglês ainda — pelo que pude perceber.

[twitter]: https://twitter.com/hugobessaa

Essa foi uma decisão meio difícil de tomar, já que trabalho em uma empresa em que todos falam inglês, mas poucos entendem português. Quando quero compartilhar algo com eles, escrevo em inglês no blog interno ou traduzo um post como fiz.

Se você quer conseguir um trabalho fora do país, é legal compartilhar pelo menos algumas coisas em inglês. Ajuda a mostrar que sabe escrever e se expressar 😄.

**Comment by malaquiasdev** [comment]
@felipefialho  que tal reversar? 

Essa é uma coisa que acho que vai acabar me atrapalhando um pouco, pq realmente precisa dedicação. Mas eu tava pensando em fazer um revezamento semanal: 1 semana em inglês e 1 semana em pt-br

**Comment by felipefialho** [comment]
@hugobessaa Nossa cara, exato!

O alcance no Brasil quando você "migra" de pt-br para inglês cai demais, e estando aqui no Brasil, não acho uma boa prática (especialmente tuites e etc).

É uma questão bem complexa e dificil formar uma opinião.

**Comment by andre-mendes** [comment]
Como o @hugobessaa comentou, gostaria de focar na comunidade local, então acho válido seguir o que o @felipefialho  comentou.... se o conteudo já foi muito abordado na comunidade estrangeira e pouco na BR, escreva em PT-BR, mas sempre escrever algo em ingles pra não se 'excluir' da comunidade gringa :D

**Comment by hugooliveirad** [comment]
Eu comecei a escrever em inglês quando quase todas as pessoas que eu seguia falavam inglês e quase todas as conversas que eu tinha no twitter eram em inglês.

Comecei a seguir mais brasileiros agora 😄.

**Comment by GuilhermeOmetto** [comment]
Eu me encaixo nesse perfil da comunidade local, não tenho grandes conhecimentos em inglês. Dessa forma sempre busco conteúdo em pt-br e valorizo quem escreve para os seus, me refiro a se queremos fortalecer a comunidade local temos que corresponder de acordo "falar a mesma língua". 

Faço pesquisas em inglês como todos mas claro com dificuldades por não ter o domínio da língua, vendo os comentários anteriores acho correto sim você escrever em inglês se quer ter uma visibilidade maior ou participar da comunidade global. Mas se tratando de alavancar a comunidade local pt-br sem dúvidas, a migração de cada dev para o inglês vêm com o tempo com sua evolução é gradativo.

**Comment by obetomuniz** [comment]
@felipefialho  Compartilho da sua opinião na descrição, pois tento sempre analisar a qualidade e o volume do conteúdo gerado localmente. Logo minhas recomendações são:

- Se o conteúdo não existe em PT_BR ou EN, faço em ambas as linguas.
- Se o conteúdo não existe ou existe em PT_BR, mas sem riqueza de detalhes ou profundidade, porém já é muito comentado em EN, faço em PT_BR apenas.
- Se o  conteúdo existe  PT_BR e EN, mesmo que com qualidade, ainda assim escrevo algo, mas no caso, tento escrever colocando minha perspectiva sobre o assunto, para justamente sair da mesmice e oferecer diferentes perspectivas de um determinado assunto que talvez já esteja saturado, mas que pessoalmente quero explorar um pouco também.

**Comment by pedrobritto** [comment]
Pergunta interessante.

Atualmente mantenho um repo pessoal bem simples com algumas notas, códigos de exemplo e 'mini tutoriais' na wiki do repo sobre qualquer coisa nova, útil e que foi difícil pra eu aprender (ex: teoria e código para criar uma grid responsiva em flexbox). 

Fiquei em dúvida se escreveria ele em inglês ou em português e meu primeiro pensamento foi escrever em inglês, mas vi que esse conteúdo poderia ajudar iniciantes brasileiros como eu. Isso porque quando procuro sobre algum assunto novo, pesquiso nos dois idiomas e a quantidade de artigos sobre o assunto em português é normalmente muito baixa e com qualidade um pouco inferior ao conteúdo em inglês. Mas esse é o meu caso.

Eu acho a sugestão do @hugobessaa muito boa e perfeitamente balanceada.

**Comment by matheusml** [comment]
Excelente tópico, talvez a minha história consiga esclarecer alguns pontos e tirar algumas dúvidas.

## Antes
Eu comecei a escrever em 2012 em inglês, e mantive essa pegada até o final de 2014.
Apesar de eu saber escrever bem, o meu _blog_ era **pouquíssimo lido**.

Infelizmente só fui descobrir o motivo tarde: se você quer escrever em inglês, você vai enfrentar uma **concorrência muito qualificada**, **altamente atualizada** e **com recursos pra divulgar os _posts_ muito melhores do que você**.

Além disso, eu simplesmente não tinha um **nicho**. 
Eu escrevia basicamente sobre as coisas novas que eu aprendia no trabalho. Ou seja, não tinha uma **temática** para que os leitores pudessem se identificar. E **sem essa identificação eles não retornavam pro meu _blog_** e pior: não **divulgavam** meus _posts_.

## Agora
Até que em 2015 resolvi mudar totalmente a minha abordagem em relação ao _blog_.

#### Escrever em português
Infelizmente o mercado brasileiro é atrasado em relação ao mercado americano. 
Exemplo: enquanto AngularJS começou a ficar na moda lá fora em 2012, aqui só começou a ganhar tração em 2014.

Resolvi então **mudar a minha escrita para o português** e tentar diminuir esse _gap_ o máximo possível.
A ideia era observar as tendências lá de fora, tentar separar o que ia dar certo e o que não ia, ler o máximo possível sobre o assunto, e tentar resumir (em português) pros leitores daqui.

Um dos meus _posts_ mais lidos até hoje, [Como trabalhar com AngularJS - O Guia Absolutamente Completo](https://medium.com/concrete-solutions/como-trabalhar-com-angularjs-o-guia-absolutamente-completo-4def67b41e20#.5dsrvziuc), seguiu exatamente essa tática.
Eu observei que o Angular estava ganhando corpo, mas existia pouquíssimo conteúdo em português.
Ou seja, encontrei um **mercado** e supri a demanda.

#### Encontrar um nicho
Isso é extremamente importante.
Se o leitor tiver uma ideia dos temas que você geralmente escreve, é muito mais provável que ele siga o seu _blog_ de perto, compartilhe seus _posts_ e talvez até compre produtos que você queira vender.

A temática [do meu _blog_](https://medium.com/@matheusml), por exemplo é:
- JavaScript
- Programação Funcional

E nesse ano, estou querendo adicionar mais uma categoria:
- Progressive Web Apps

Com isso, o leitor já sabe o que esperar de mim. E principalmente: ele sabe o que **não** esperar.

#### Divugalção
Se antes a minha divulgação era praticamente nula, agora é um dos pontos que mais dou ênfase.
E justamente por ter escolhido fazer o _switch_ pra português, isso se tornou muito mais fácil.

Quando eu não tinha muito nome e nem seguidores, eu simplesmente **divulgava nos grupos de Facebook** brasileiros que tem alguma relação com o tema.

Além disso, eu tentava palestrar, criar _workshops_ e eventos no geral, focados nos temas sobre o qual eu escrevo. Com isso, aos poucos, eu criava um nome e gerava mais seguidores.

Depois, criei uma [_newsletter_ pessoal](http://matheuslima.us12.list-manage.com/subscribe?u=10a2683fc9e0f0f9ff2c6d9af&id=4211beeb8d) para que as pessoas comecem a acompanhar meus _posts_ mais de perto.

Com isso, passou a ser possível divulgar meus _posts_ também no [Twitter](https://twitter.com/matheusml), porque agora já consigo ter um número de seguidores/_retweets_ razoáveis.

E por último, com o foco que eu dou na **qualidade** dos _posts_, fui convidado para ser _writer_ do [Tableless](https://medium.com/tableless). E agora meus _posts_ tem uma visibilidade muito acima do normal.

#### Conclusão
Com todas essas medidas:
- Começar a escrever em português
- Encontrar uma temática para o meu _blog_
- Achar um nicho de leitores
- Focar na divulgação

Eu consegui a média de **4,4mil leituras por mês** no meu _blog_ no ano de 2016

**Comment by felipefialho** [comment]
@matheusml Sabe que sou leitor assiduo do seu blog né?

Fácil entre os melhores brasileiros na ativa.

Mantém a pegada mano 👏 👏 👏

**Comment by hugocarreira** [comment]
Usando o que o @felipefialho  falou...
Também sou leitor do seu blog, @matheusml, à pouco tempo, achei ele procurando conteúdo do AngularJS e gostei muito, parabéns! E uma observaçãozinha, acabei de descobrir que você é meu conterrâneo na cidade sorriso, irado...


Adicionando ao tópico, tenho um blog porem não sou muito ativo, penso em voltar à escrever todos os dias, mas falta algo que me motive.

Eu particularmente, prefiro escrever em PT-BR, creio que é melhor pois ajuda a movimentar a comunidade br, como o @matheusml disse, existe uma falta de artigos bons em português (não que o pessoal escreva mal), mas a quantidade de conteúdo em inglês é bem maior.

Mas também creio que seja algo pessoal do escritor, caso ele queira atingir um público maior é obvio que em inglês ele vai conseguir isso..

**Comment by erickpatrick** [comment]
Tópico muito bom. 

Embora não esteja escrevendo meus próprios artigos, tenho feito muitas traduções para o português ([muitas mesmo](https://tutsplus.com/authors/erick-patrick)). Sempre senti que a comunidade brasileira precisasse de mais conteúdo em nosso idioma.

Particularmente, acredito que todo desenvolvedor precise ter um bom/ótimo nível de inglês, contudo, devemos fazer, na medida possível, um esforço para criar conteúdo em nosso idioma também. 

Se quisermos incentivar novas gerações (crianças de 6, 8... 10 anos), que, provavelmente, ainda não terão um nível bom de inglês para entender as nuances do desenvolvimento; pessoas que não tem ou não tiveram as mesmas oportunidades; pessoas que estão tentando migrar de outras áreas; ou que estão tentando se "reencontrar" dentro da área de TI, ter conteúdo de qualidade em português é um ótimo atrativo/estimulante.

Por exemplo, tenho traduzido porque, além de ser um jeito de aprender novas tecnologias/ferramentas, coloca meu inglês em prática e acredito trazer conteúdo bacana para nosso idioma.

Mas, será que é preciso mesmo criar conteúdo em inglês para chamar atenção de fora? Ou será que crescer primeiro no "mercado interno", chamando atenção de gente que tem contatos fora e levando nosso conteúdo "traduzido" para esse povo também dá certo?

**Comment by thulioph** [comment]
Sempre fui fã e optei por conteúdos escritos em português (pt-br).

A disparidade de conteúdo em inglês na internet é muito maior, esse é o principal motivo que me faz escrever na minha língua nativa. 

Tradução também é uma coisa que eu curto fazer, e em relação a "chamar atenção dos gringos", apenas faça um bom trabalho, um dia a sua hora chega. 

Se ao menos 1 dos n artigos que você lê em inglês todo dia, você traduzir, já vai tá colaborando pra comunidade local e melhorando seu inglês.

**Comment by leandroruel** [comment]
eu acho ótimo fazer em português para a galera conseguir buscar no google quando precisar, em pt-br é bastante difícil achar artigos atualizados.

eis um exemplo, blog das antigas e sempre postando em pt, e traduzindo artigos da gringa: http://desenvolvimentoparaweb.com

**Comment by thierryrene** [comment]
Eu sempre preferi conteúdo em inglês, pelo padrão das techs e pela facilidade em encontrar conteúdo, mas como o erick, eu também traduzo artigos no Tuts+, e lá tem sim muito conteúdo atualizado em PT-BR.

**Comment by erickpatrick** [comment]
"Revivendo o tópico". 

Trago meu [repositório com listas de termos de TI/Design](https://github.com/erickpatrick/termos-ti-design-pt-br) com traduções para o português que venho usando nas minhas traduções. Estava tentando atualizar aos poucos e ter todas os termos já lá, mas resolvi liberar logo, já que o pessoal que traduz pro Tuts+ tem usado com frequência. Com mais e mais frequências vocês verão esses termos (parece presunção, eu sei, mas...), não só vindos de mim e minhas traduções mas de mais pessoas, já que, como disse, tem mais gente do Tuts+ traduzindo, usando e espalhando.

Minha ideia é criar um hub/dicionário com os termos e traduções. Se tiver algum sugestão de termo, abra uma requisição de adição (PR) ou uma requisição via tópicos (issues) caso busque algum termo e não tenha lá ainda. Espero a ajuda de vocês, seja contribuindo diretamente (PR) ou indiretamente (compartilhando o link).

Logo colocarei mais listas da "sopa de letrinha" lá, só preciso ajustar melhor meu tempo entre o trabalho, traduções e casa/família =)

**Comment by rubenmarcus** [comment]
@erickpatrick  muito obrigado pelo seu depoimento estou referenciando [isso ](https://github.com/frontendbr/forum/issues/1400) , estou com perfil na [MDN ](https://developer.mozilla.org/pt-BR/profiles/rubenmarcus), por enquanto so traduzi 2 artigos, mas quero traduzir mais, ja tem 5 na mira aqui kkk
